### PR TITLE
[state sync] more logging + counters

### PR DESCRIPTION
**Pull request by christinacelee**
## Motivation

(Write your motivation for proposed changes here.)

### Have you read the [Contributing Guidelines on pull requests](https://github.com/libra/libra/blob/master/CONTRIBUTING.md#pull-requests)?

(Write your answer here.)

## Test Plan

(Share your test plan here. If you changed code, please provide us with clear instructions for verifying that your changes work.)

## Related PRs

(If this PR adds or changes functionality, please take some time to update the docs at https://github.com/libra/libra/developers.libra.org, and link to your PR here.)

### CODE REVIEW

File: state-synchronizer/src/request_manager.rs [hunk 1]
```diff
@@ -252,10 +253,10 @@ impl RequestManager {
         }
 
         let req_info = self.add_request(req.known_version, peers.clone());
-        debug!(
-            "[state sync] request next chunk - chunk req info: {:?}",
-            req_info
-        );
+        debug!(log
+            .clone()
```
Comment (bmwill) [inline state-synchronizer/src/request_manager.rs:257]: Does this actually need to be cloned?

**Review by bmwill — APPROVED** [review]

File: state-synchronizer/src/request_manager.rs [hunk 2]
```diff
@@ -439,7 +447,13 @@ impl RequestManager {
                 self.multicast_level + 1,
                 self.upstream_config.upstream_count(),
             );
-            debug!("[state sync] multicast timeout occurred for version {}, update multicast level from {} to {}", version, prev_multicast_level, self.multicast_level);
+            debug!(
```
Comment (davidiw) [inline state-synchronizer/src/request_manager.rs:450]: should this be an info? probably should just open up state-sync and write comments there...

File: state-synchronizer/src/request_manager.rs (hunk 1 above)
Comment (christinacelee) [reply state-synchronizer/src/request_manager.rs:257]: yeah, it potentially gets moved later in the code

**Pull request closed without merging**